### PR TITLE
add landing page for cnd config practices workshop

### PR DESCRIPTION
**Pull request by billkable**
Request addition of landing page for the Cloud Native Fundamentals Configuration Practices workshop.

### CODE REVIEW

**Comment by tybritten** [comment]
LGTM. Let me know when you have a workshop ready to deploy on the cluster

File: content/workshops/cnd-config-practices.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+This is a workshop for developers and application operators
+new to cloud native concepts who wish to learn the basics of
+handling environmental configuration separately from a web application
+codebase using modern orchestration platform tools.
+
+After completing the workshop, you will be able to:
+
+-   Explain configuration requirements of Cloud Native applications
```
Comment (zmb3) [inline content/workshops/cnd-config-practices.md:29]: Will this properly render as a `<ul>`? 

I've never seen a markdown list with so much whitespace between the `-` and the text.

**Comment by billkable** [comment]
> LGTM. Let me know when you have a workshop ready to deploy on the cluster

@tybritten I will reach on on Slack later today - we have a video playlist to upload, but otherwise the the staging and production images are in GHCR.  Production is here: `ghcr.io/platform-acceleration-lab/cnd-config-practices:production`

File: content/workshops/cnd-config-practices.md (hunk 1 above)
Comment (billkable) [reply content/workshops/cnd-config-practices.md:29]: @zmb3 Sorry, I almost missed your comment.  

Much of the markdown we authored in the previous master class and product content had liberal whitespace, and line breaks on phrase or sentence separators (for hopefully better diffs between commits).

If TDC has a formal set of maintainer formatting guidelines, if you point me to those, I'll follow for future content edits.

**Review by tybritten — APPROVED** [review]

**Pull request merged**